packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/accessories/shipping/telephony-line-phone-accessories-shipping.service.js

```diff
@@ -0,0 +1,31 @@
+import filter from 'lodash/filter';
+import get from 'lodash/get';
+import groupBy from 'lodash/groupBy';
+import map from 'lodash/map';
+
+export default function filterContact(contacts) {
+  return filter(
+    map(
+      groupBy(contacts, (contact) => {
+        // group contact to detect contact that are the same
+        const contactCopy = {
+          lastName: contact.lastName,
+          firstName: contact.firstName,
+        };
+        if (contact.address) {
+          contactCopy.address = {
+            country: contact.address.country,
+            line1: contact.address.line1,
+            zip: contact.address.zip,
+            city: contact.address.city,
+          };
+        }
+        return JSON.stringify(contactCopy);
+      }),
+      (groups) => groups[0],
+    ),
+    (contact) =>
+      get(contact, 'address') &&
+      ['BE', 'FR', 'CH'].indexOf(contact.address.country) > -1,
```

Comment:
Same here for:
```suggestion
      ['BE', 'FR', 'CH'].includes(contact.address.country),
```

Also the array should have a name to ease code reading in the future